buildsupport/toolchains/toolchain-mingw64-linux-cross.cmake

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2024 Johannes Kepler University Linz
+
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Alois Zoitl - Initial contribution.
+###############################################################################
+
+SET(CMAKE_SYSTEM_NAME Windows)
+SET(TOOLCHAIN_PREFIX x86_64-w64-mingw32)
+
+SET(CMAKE_C_COMPILER ${TOOLCHAIN_BIN}/arm-none-eabi-gcc)
```

Comment:
I guess lines 17 and 18 are not needed as the variables are modified again in 20 and 21?

Reply:
I assume that they are leftovers, as they also target arm systems, which I guess is not the intended target system.

As @ernstblechaPT please remove them, if we am correct

Reply:
you both are correct. Forgot to remove them. Will do that now.